app/views/services/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<head>
+    <h1>Services List</h1>
+  </head>
+<td><a href= <%= new_service_path %> class = "btn btn-success">New Service</a></td>
```

Comment:
Think about consistency when you're writing code (and templates). Here you have a lot of inconsistency, which makes it harder to read and understand.

Things that are inconsistent here:
* spacing around the attribute name, `=` and attribute value (`href= <%...`, `class = "btn...`)
* quotes around attribute values (the class attributes have `"` characters, the `href` does not)

Reply:
so... by that I'm referring to consistency between attributes. You've got that consistent between `href` already, but not between the different attributes within the tag.

Forgive if I'm going over things that are obvious / you know...

In an example html tag:
`<a href="link" class="cls1 cls2">click here</a>`
there are a few sections.
The whole thing is called the `tag` and in this case it is an `a` or `anchor` (to give it the full name) tag.
`href="link"` and `class="cls1 cls2` are attributes
`href` and `class` are attribute names
`link` and `cls1 cls2` are attribute values 
`click here` is the html tag's content
`</a>` is the "closing tag"

To make this a valid html tag, an attribute value must either:
* not be present (in which case it is interpreted as having a value of `true` if the attribute name is present, or `false` if the attribute name is not present
* `true` or `false` (without any quotes)
* a number (without quotes)
* a string (with quotes)

e.g. `<a href=http://foo.com/bar>` is not valid, but  `<a href="http://foo.com/bar">` is valid.

Now, there is some magic in browsers where most modern browsers can figure some things out. In the case of a `href` attribute on an `a` tag, the browser expects a link as the value, so it will try to figure out what you've provided.... and often be successful. But not always.

In your code, you're consistently adding the quotes around the `class` attribute values, but not around the `href` attribute values.

To be consistent (and also more correct), I would expect something like:
`<a href="<%= new_service_path %>" class="btn btn-success">New Service</a>`